server/src/routes/course/taskVerification.ts

```diff
@@ -186,9 +203,10 @@ type SelfEducationVerificationParams = {
 type SelfEducationAttributes = {
   public: {
     maxAttemptsNumber: number;
-    strictAttemptsMode?: boolean;
     numberOfQuestions: number;
     tresholdPercentage: number;
+    strictAttemptsMode?: boolean;
+    oneAttemptPerNumberOfHours?: number;
```

Comment:
what do you think about having more generic solution and set min amount of ms or seconds between runs?

e.g.
```typescript
attemptsWaitTimeSeconds: 86400; // 1 hour
```

Reply:
But you can set 0.1 hours, 0.5 hours, etc. I think it's more human friendly than ms or seconds.
And the current variable name is more self-descriptable.